pkg/config/datastore/loader.go

```diff
@@ -71,6 +71,7 @@ func buildDataLoader(viperCfg func(v *viper.Viper)) config.DataStoreLoaderFunc {
 		}
 
 		cfg.Data = &data
+		cfg.ConfigFilePath = v.ConfigFileUsed()
```

Comment:
I am persisting this so that when I check file permissions I know which file in PORTER_HOME was the loaded config file, since it could have been config.toml/json/yaml.